scapy/dadict.py

```diff
@@ -25,7 +25,7 @@ class DADict_Exception(Scapy_Exception):
 class DADict:
     def __init__(self, _name="DADict", **kargs):
         self._name=_name
-        self.__dict__.update(kargs)
+        self._update(kargs)
```

Comment:
Renaming it `update` does not work (it sill call the __dict__ one).
__dict__.update is a readonly value, so impossible to overwrite...